include/psp2common/kernel/processmgr.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \kernelgroup{SceProcessmgr}
+ * \usage{psp2common/kernel/processmgr.h}
+ */
+
+#ifndef _PSP2COMMON_KERNEL_PROCESSMGR_H_
+#define _PSP2COMMON_KERNEL_PROCESSMGR_H_
+
+#include <vitasdk/build_utils.h>
+#include <psp2common/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+typedef SceUInt32 SceKernelProcessType;
+
+typedef SceUInt64 SceKernelClock;
+typedef SceUInt32 SceKernelTime; 
+
+typedef enum SceKernelProcessPrioritySystem : SceInt32 {
+	SCE_KERNEL_PROCESS_PRIORITY_SYSTEM_HIGH     = 32,
+	SCE_KERNEL_PROCESS_PRIORITY_SYSTEM_DEFAULT  = 96,
+	SCE_KERNEL_PROCESS_PRIORITY_SYSTEM_LOW      = 159
+} SceKernelProcessPrioritySystem;
+
+typedef enum SceKernelProcessPriorityUser : SceInt32 {
+	SCE_KERNEL_PROCESS_PRIORITY_USER_HIGH       = 64,
+	SCE_KERNEL_PROCESS_PRIORITY_USER_DEFAULT    = 96,
+	SCE_KERNEL_PROCESS_PRIORITY_USER_LOW        = 127
+} SceKernelProcessPriorityUser;
```

Comment:
they are also required a size check or we can remove the size check for the `SceKErnelPowerTickType`
(because we already marked that these enum types are `SceInt32` :))

Reply:
Added check.